src/node/progress_tracker.h

```diff
@@ -710,13 +635,146 @@ namespace ccf
       return true;
     }
 
+    Nonce get_my_nonce(kv::TxID tx_id)
```

Comment:
get_node_nonce, get_local_node_nonce? I don't believe we use "my" anywhere else, and we certainly don't in the external APIs.